lib/core/include/irods/library_features.h

```diff
@@ -49,4 +49,9 @@
 /// \since 4.3.1
 #define IRODS_HAS_API_ENDPOINT_CHECK_AUTH_CREDENTIALS                        202307L
 
+/// Defined if the development library supports #rc_replica_truncate.
+///
+/// \since 4.3.2
+#define IRODS_HAS_API_ENDPOINT_REPLICA_TRUNCATE                              202403L
```

Comment:
we're going to get 4.3.2 out in march?  or... we're just tracking when added?

Reply:
This is tracking when fully implemented and added.

Reply:
okay, so no alignment with release date.    carry on.